src/components/action-buttons.component.tsx

```diff
@@ -55,14 +58,31 @@ const ActionButtons: React.FC<ActionButtonsProps> = ({
     medicationRequestStatus === MedicationRequestStatus.active &&
     mostRecentMedicationDispenseStatus !== MedicationDispenseStatus.declined;
 
+  // eslint-disable-next-line no-console
+  console.log(patient, "patient");
+
+  const DesignName = () => {
+    const styles = { paddingLeft: "20px", color: "red" };
+    if (patient) {
+      return (
+        <span style={styles}>{`${patient.name} - #${patient.identifier}`}</span>
+      );
+    }
+
+    return <span style={styles}>--- --</span>;
+  };
```

Comment:
Based on the designs, the patient header would be a better option here

Reply:
sorry to ask, but what do you mean by patient header @pirupius?

Reply:
is it like a carbon component?

Reply:
Hey @Jexsie I've just cross checked and it wasn't included in the designs for the `Dispense Form` but it looks like this 

<img width="408" alt="Screenshot 2023-04-27 at 13 07 56" src="https://user-images.githubusercontent.com/15266028/234831330-4f4d8b88-cb88-46c3-a6f1-9e0dbf97b02f.png">


https://app.zeplin.io/project/61af2b325f2c328f972eafbd/screen/62135fdbc9e1e71095e2489e

Reply:
@pirupius, I had followed this design provided by Fion in the ticket
![image](https://user-images.githubusercontent.com/104269786/234833575-47dd8708-01a9-4c99-bf8b-110bb7d89816.png)